rss.py

```
@@ -0,0 +1,93 @@
+        lower_text = text.lower()
+
+        if i == 0 and "getcomics" in lower_text:
+            continue
+        if i == len(paragraphs) - 1 and (
+            "the post" in lower_text or "appeared first on" in lower_text
+        ):
+            continue
+
+        if is_metadata_paragraph(p):
+            continue
+        
```

Flake8 (7.2.0)

[warning] blank line contains whitespace

(W293)